fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -5031,6 +5033,11 @@ public LogicalPlan visitShowDelete(ShowDeleteContext ctx) {
         return new ShowDeleteCommand(dbName);
     }
 
+    @Override
+    public LogicalPlan visitShowStoragePolicy(ShowStoragePolicyContext ctx) {
+        return new ShowStoragePolicyCommand(ctx.identifierOrText().getText());
```

Comment:
policy name is optional, please check if policy exists

Reply:
done - please review